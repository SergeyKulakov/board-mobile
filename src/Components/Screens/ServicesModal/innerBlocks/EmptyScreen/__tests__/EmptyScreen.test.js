import React from 'react'
import { shallow, mount } from 'enzyme'
import { expect as chaiExpect } from 'chai'

import 'jest-styled-components'

import { EmptyScreen } from '../index'

describe('ServicesModal.EmptyScreen', () => {
  const defaultProps = {
    value: '',
  }

  it('render snapshot', () => {
    const wrapper = shallow(<EmptyScreen {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })

  // it('render props.value', () => {
  //   const wrapper = shallow(<Component {...defaultProps} />)
  //
  //   console.debug(wrapper.find('[testID="text"]').children)
  //   chaiExpect(wrapper.find('[testID="text"]').children).to.be.equal('server error')
  //
  //   const nextValue = 'test'
  //   wrapper.setProps({ value: nextValue })
  //
  //   chaiExpect(wrapper.find('[testID="text"]').children).to.be.equal(`${nextValue} not found`)
  // })
  //
  // it('should be icon search', () => {
  //   const wrapper = mount(<Component {...defaultProps} />)
  //
  //
  // })
})
