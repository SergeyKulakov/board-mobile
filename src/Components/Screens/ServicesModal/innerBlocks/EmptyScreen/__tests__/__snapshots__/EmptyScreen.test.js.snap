// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServicesModal.EmptyScreen render snapshot 1`] = `
<Styled(View)>
  <Icon
    color="#69696a"
    name="search"
    size={40}
  />
  <Styled(Text)
    testID="text"
  >
    server error
  </Styled(Text)>
</Styled(View)>
`;
